apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -1283,7 +1283,7 @@ export class PretixPipeline implements BasePipeline {
       ticketId: atom.id,
       eventId: atom.eventId,
       productId: atom.productId,
-      timestampConsumed: atom.timestampConsumed?.getTime() ?? 0,
+      timestampConsumed: atom.timestampConsumed?.getTime?.() ?? 0,
```

Comment:
not sure how this type of thing is not caught by the compiler - `strictNullChecks` is not catching issues like this which is a problem. someone should look into this.